lib/plugins/features/movies/controller.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const Movie = require('../../../models/movie');
+
+// defining code that is shared accross different files. Important to include so that the funciton or the variable can be used outside of this file
```

Comment:
Nit: Remove note comments from the final PR. Generally we don't tend to comment too much of our production code unless it really needs clarification.